examples/example-gcp-observability/src/main/java/io/grpc/examples/observabilityHelloWorld/ObservabilityHelloWorldClient.java

```diff
@@ -0,0 +1,114 @@
+    // 'channel' here is a Channel, not a ManagedChannel, so it is not this code's responsibility to
+    // shut it down.
+
+    // Passing Channels to code makes code easier to test and makes it easier to reuse Channels.
+    blockingStub = GreeterGrpc.newBlockingStub(channel);
+  }
+
+  /** Say hello to server. */
+  public void greet(String name) {
+    logger.info("Will try to greet " + name + " ...");
+    HelloRequest request = HelloRequest.newBuilder().setName(name).build();
+    HelloReply response;
+    try {
+      response = blockingStub.sayHello(request);
+    } catch (StatusRuntimeException e) {
+      logger.log(Level.WARNING, "RPC failed: {0}", e.getStatus());
+      return;
+    }
+    logger.info("Greeting: " + response.getMessage());
+  }
+
+  /**
+   * Greet server. If provided, the first element of {@code args} is the name to use in the
+   * greeting. The second argument is the target server.
+   */
+  public static void main(String[] args) throws Exception {
+    String user = "world";
+    // Access a service running on the local machine on port 50051
+    String target = "localhost:50051";
+    // Allow passing in the user and target strings as command line arguments
+    if (args.length > 0) {
+      if ("--help".equals(args[0])) {
+        System.err.println("Usage: [name [target]]");
+        System.err.println("");
+        System.err.println("  name    The name you wish to be greeted by. Defaults to " + user);
+        System.err.println("  target  The server to connect to. Defaults to " + target);
+        System.exit(1);
+      }
+      user = args[0];
+    }
+    if (args.length > 1) {
+      target = args[1];
+    }
+
+    // Initialize observability
+    try (GcpObservability observability = GcpObservability.grpcInit()) {
+      // Create a communication channel to the server, known as a Channel. Channels are thread-safe
+      // and reusable. It is common to create channels at the beginning of your application and reuse
+      // them until the application shuts down.
+      //
+      // For the example we use plaintext insecure credentials to avoid needing TLS certificates. To
+      // use TLS, use TlsChannelCredentials instead.
+      ManagedChannel channel = Grpc.newChannelBuilder(target, InsecureChannelCredentials.create())
+          .build();
+      try {
+        ObservabilityHelloWorldClient client = new ObservabilityHelloWorldClient(channel);
+        client.greet(user);
+      } finally {
+        // ManagedChannels use resources like threads and TCP connections. To prevent leaking these
+        // resources the channel should be shut down when it will no longer be used. If it may be used
+        // again leave it running.
+        channel.shutdownNow().awaitTermination(5, TimeUnit.SECONDS);
+      }
+      System.out.println(
+          String.format(
+              "Sleeping %d seconds before shutdown to ensure all records are flushed.",
+              EXPORT_INTERVAL));
```

Comment:
This only applies to logging or the other signals also? In that case: can we rename `EXPORT_INTERVAL` to `LOGGING_EXPORT_INTERVAL` ?

And another "nice-to-do" but not strictly required thing is do this only if logging is enabled in your observability config. Not sure if others will agree with this suggestion

Reply:
Removed sleep from example, and moving to `observability.close()` instead (follow-up PR).